apps/web/src/pages/studio-onboarding/success.tsx

```diff
@@ -0,0 +1,82 @@
+import { Title, Text } from '@novu/novui';
+import { css } from '@novu/novui/css';
+import { VStack } from '@novu/novui/jsx';
+import { useQuery } from '@tanstack/react-query';
+import { useMemo } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { getActivityList } from '../../api/activity';
+import { ExecutionDetailsAccordion } from '../../components/execution-detail/ExecutionDetailsAccordion';
+import { ROUTES } from '../../constants/routes';
+import { Footer } from './components/Footer';
+import { Header } from './components/Header';
+
+export const StudioOnboardingSuccess = () => {
+  const { data } = useQuery<{ data: any[]; hasMore: boolean; pageSize: number }>(['activitiesList', 0], () =>
+    getActivityList(0, undefined)
+  );
+
+  const item = useMemo(() => {
+    if (!data) {
+      return null;
+    }
+
+    return data.data.at(-1);
+  }, [data]);
+
+  const identifier = useMemo(() => {
+    if (item === null) {
+      return undefined;
+    }
+
+    return item?.trigger?.identifier;
+  }, [item]);
+
+  const email = useMemo(() => {
+    if (item === null) {
+      return null;
+    }
+
+    return item.subscriber.email;
+  }, [item]);
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
💭 **thought (non-blocking):** This seems like a similar "chain" of memos to `preview.tsx` where could extract the queries to a single async function outside of React.‏